src/status_im/chat/models/message_content.cljs

```diff
@@ -87,12 +88,15 @@
   Value for each key is sequence of tuples representing ranges in original
   `:text` content. "
   [{:keys [text] :as content}]
+  ; (log/debug "enrich-content text = " text)
+  ; (log/debug "enrich-content content = " content)
   (let [[_ metadata] (reduce (fn [[text metadata] [type regex]]
                                (if-let [matches (query-regex regex text)]
                                  [(clear-ranges matches text) (assoc metadata type matches)]
                                  [text metadata]))
                              [text {}]
                              (into stylings actions))]
+    ; (log/debug "enrich-content metadata = " metadata)
```

Comment:
remove commented-out code

Reply:
oh yeah forgot to remove my debug messages